apps/AppTracking.js

```diff
@@ -18,6 +18,8 @@ var AppTracking = function() {
   this.applicationVersion = undefined;
   this.sourceSDK = undefined;
   this.sourceSDKVersion = undefined;
+  this.dansBadVar = undefined;
+  this.should_be_camel_case = undefined
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined.](https://app.codacy.com/manual/dan-codacy/remote-pay-cloud-api/pullRequest?prid=5044103)